helm-chart/Chart.yaml

```diff
@@ -2,6 +2,6 @@ apiVersion: v2
 name: hedera-json-rpc-relay
 description: Helm chart deployment of the hashgraph/hedera-json-rpc-relay
 type: application
-version: 0.8.0-SNAPSHOT
-appVersion: "0.8.0-SNAPSHOT"
+version: 0.1.1
+appVersion: "0.7.0"
```

Comment:
Something seems wrong with either the PR or with the release automation to have such version skew. I see in the helm repo the last is `0.1.0` with `0.7.0` app. I think the `appVersion=0.8.0-SNAPSHOT` is correct and should be reverted since this is main and `0.7.0` has been released already. 

But the `version=0.8.0-SNAPSHOT` may or may not be correct. Depends if the relay is moving to synchronize appVersion and version like the mirror node did or whether it's possible for the release lifecycle to vary for the two artifacts.

Reply:
version & appVersion should be synchronized. 
`0.8.0-SNAPSHOT` is the right value.

Is this just a bad merge?

Reply:
I changed this b/c I didn't see any release of 0.8.0-SNAPSHOT in the release list.

Reply:
Ahh you're right.
The SNAPSHOTS refer to the latest main code.
Deployment should use GAd versions under releases